demoapp/src/androidTest/java/me/uport/sdk/demoapp/SelectiveDisclosureFlowTest.kt

```diff
@@ -15,17 +15,22 @@ import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
 import android.support.test.espresso.matcher.ViewMatchers.withId
 import me.uport.sdk.demoapp.request_flows.uPortLoginActivity
 import me.uport.sdk.transport.RequestDispatchActivity
+import org.hamcrest.Matchers.allOf
 import org.junit.After
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import ro.mirceanistor.testutil.ViewMatcherIdlingRule
 
 
-class uPortLoginTest {
+class SelectiveDisclosureFlowTest {
 
     @get:Rule
     val intentsTestRule = IntentsTestRule(uPortLoginActivity::class.java)
 
+    @get:Rule
+    val viewMatcherIdlingRule = ViewMatcherIdlingRule(allOf(withId(R.id.progress), isDisplayed()))
```

Comment:
viewMatcherIdlingRule doesn't seem to ever be used. Is it necessary to have a variable for this?